oat-sa/extension-tao-itemqti-pci
# MS-988 feature: prevent recording and resetting in review mode

Ticket: https://oat-sa.atlassian.net/browse/MS-988

Contains: 

- Review mode config option for audio PCI
-  Recording is disabled in review mode
- Resetting is disabled in review mode

How to test:

- Checkout the branch
- run the migration:

```
php index.php '\oat\tao\scripts\tools\Migrations' -c execute -v 'oat\qtiItemPci\migrations\Version202103240929581465_qtiItemPci'
```
- Create an item with audio PCI and give a test
- Check the audio PCI interaction in delivery mode (it should contain all the controls)
- Check the audio PCI interaction in review mode (it should only contain play and stop)

Requires:

- [x] https://github.com/oat-sa/tao-item-runner-qti-fe/pull/161
- [x] https://github.com/oat-sa/extension-tao-itemqti/pull/1665
